docs/docs-content/clusters/cluster-management/ssh-keys/ssh-usernames.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_label: "SSH Usernames"
+title: "SSH Usernames"
+description:
+  "A list of the Secure Shell (SSH) usernames created on Kubernetes nodes for each provider and operating system pack
+  available in Palette."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 11
+---
+
+This page lists the Secure Shell (SSH) user names created on Kubernetes nodes, which vary by provider and operating
+system.
+
+These user names are relevant when using [SSH key pairs](./ssh-keys.md) with your Palette host clusters.
+
+## Public Cloud
+
+### Infrastructure Provider
+
+| Provider       | Operating System Pack | SSH Username |
+| -------------- | --------------------- | ------------ |
+| **AWS IaaS**   | **Ubuntu**            | `ubuntu`     |
+| **AWS IaaS**   | **CentOS**            | `centos`     |
+| **Azure IaaS** | **Ubuntu**            | `ubuntu`     |
+| **Azure IaaS** | **CentOS**            | `centos`     |
+| **GCP IaaS**   | **Ubuntu**            | `ubuntu`     |
+| **GCP IaaS**   | **CentOS**            | `centos`     |
+
+### Managed Kubernetes
+
+| Provider      | Operating System Pack          | SSH Username |
+| ------------- | ------------------------------ | ------------ |
+| **AWS EKS**   | **Amazon EKS optimized Linux** | `ec2-user`   |
+| **Azure AKS** | **Linux**                      | `azureuser`  |
+| **GCP GKE**   | **Container-Optimized OS**     | N/A          |
+
+#### SSH Configuration on GCP GKE
+
+It is not possible to configure GCP GKE clusters with an SSH key during cluster creation. If SSH access to your GCP GKE
+nodes is critical, consider one of the following options:
+
+- Configure project or node-level metadata to include your SSH key. Refer to
+  [Predefined and custom metadata keys](https://cloud.google.com/compute/docs/metadata/overview#predefined-and-custom-metadata-keys)
+  for guidance.
+
+- Enable OS Login and associate your SSH key with your IAM identity. Refer to
+  [How OS Login works](https://cloud.google.com/compute/docs/oslogin#how_os_login_works) for guidance.
+
+- Customize startup scripts or metadata to add your SSH key during node creation in a managed instance group. Refer to
+  [Using startup scripts on Linux VMs](https://cloud.google.com/compute/docs/instances/startup-scripts/linux) for
+  guidance.
+
+## Data Center
+
+| Provider           | Operating System Pack | SSH Username |
+| ------------------ | --------------------- | ------------ |
+| **MAAS**           | **Ubuntu**            | `ubuntu`     |
+| **Openstack**      | **Ubuntu**            | `ubuntu`     |
+| **VMware vSphere** | **Ubuntu**            | `ubuntu`     |
+| **VMware vSphere** | **CentOS**            | `centos`     |
+
```

Comment:
@benradstone I think we need to call out BYOOS scenarios. There are two ways to use BYOOS. 

1)Through the default kubernets image builder project, which we have documented [here](https://docs.spectrocloud.com/byoos/image-builder/build-image/).

2) For vSphere, it's a bit different as we have added some custom logic and wrapped our own solution with the kubernetes images builder project. Check this section [out](https://docs.spectrocloud.com/byoos/image-builder/build-image-vmware/).

My guess is the default SSH username is a default value in one of the Packer config files.

Reply:
It looks like this is defined in the [pack.json](https://github.com/spectrocloud/pax/blob/master/stable/os/byoi_generic/pack.json#L10) for the BYOOS pack.